.github/workflows/deploy.yml

```
@@ -0,0 +1,56 @@
+          host: ${{ secrets.EC2_HOST }}
+          username: ${{ secrets.EC2_USERNAME }}
+          key: ${{ secrets.EC2_PRIVATE_KEY }}
+          envs: APPLICATION_PROPERTIES
+          script_stop: true
+          script: |
+            rm -rf /home/ubuntu/kkukmoa-server/current
+            mkdir /home/ubuntu/kkukmoa-server/current
+            mv /home/ubuntu/kkukmoa-server/tobe/project.jar /home/ubuntu/kkukmoa-server/current/project.jar
+            cd /home/ubuntu/kkukmoa-server/current
+            sudo fuser -k -n tcp 8080 || true
+            nohup java -jar project.jar > ./output.log 2>&1 & 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)